### PR TITLE
remove remaining references to removed function of 3.x

### PR DESCRIPTION
**Pull request by pi-r-p**
Search and delete references to removed functions:
- AUTHENTICATE
- CHUNKENCODER
- CUDF
- DOC
- DOCMODE
- FROMBITS
- ISAUTHENTICATED
- MACROCONFIGSECRET
- TOBITS
- UDF
- STACKPSSECRET
- TOKENSECRET
- MAXURLFETCHCOUNT
- MAXURLFETCHSIZE
- URLFETCH

Remove unmaintained test tool to export list of functions (better use `FUNCTIONS` for inventory).

### CODE REVIEW

File: etc/conf.templates/distributed/70--extensions.conf.template [hunk 1]
```diff
@@ -67,14 +67,14 @@
 // SHM ttl in ms, default = 1 hour
 #shm.ttl = 3600000
 
-// TOKENDUMP, TOKENGEN, TOKENSECRET
+// TOKENDUMP, TOKENGEN
```
Comment (steven-gueguen) [inline etc/conf.templates/distributed/70--extensions.conf.template:70]: Please update the copyright date

File: warp10/src/main/java/io/warp10/continuum/store/Constants.java [hunk 2]
```diff
@@ -450,11 +450,6 @@ public class Constants {
    */
   public static final String TOKEN_ATTR_NOFETCH = ".nofetch";
```
Comment (steven-gueguen) [inline warp10/src/main/java/io/warp10/continuum/store/Constants.java:451]: Please update the copyright date

File: warpscript/build.gradle [hunk 3]
```diff
@@ -26,7 +26,6 @@ sourceSets {
             include 'io/warp10/script/**'
             exclude 'io/warp10/script/ScriptRunner*.*'
             include 'io/warp10/CapacityExtractorOutputStream.*'
```
Comment (steven-gueguen) [inline warpscript/build.gradle:28]: Please update the copyright date

**Review by steven-gueguen — CHANGES_REQUESTED** [review]

**Comment by pi-r-p** [comment]
Done

**Review by steven-gueguen — APPROVED** [review]

**Pull request merged**